privaterelay/locales/l10n-pontoon.toml

```diff
@@ -0,0 +1,7 @@
+# Entrypoint for community localization.
+# Edits to add content for l10n should be made in configs/pontoon.toml
+
+basepath = "."
+
+[[includes]]
+    path = "configs/pontoon.toml"
```

Comment:
This path `configs/pontoon.toml` doesn't match the directory structure `config/pontoon.toml`?

Reply:
Fixed!